### DISCUSSION TITLE
第2节课笔记(21班)

### DISCUSSION THREAD

**Discussion by mattheliu** [InternLM/Tutorial#222] [笔记]
请提交笔记链接 Please submit the link to the notes.

**Comment by JinProton** [comment]
https://zhuanlan.zhihu.com/p/677089410

**Comment by cqu-student** [comment]
https://zhuanlan.zhihu.com/p/677145690

**Comment by jayjayhust** [comment]
https://github.com/jayjayhust/InternLM-Tutorial/blob/main/Lesson2.md

**Comment by HaroldChen19** [comment]
https://github.com/HaroldChen19/InternLM-Boot-Camp/blob/master/Lesson_2/Note.md

**Comment by yunhuisong2023** [comment]
https://reflective-value-ac9.notion.site/2-21-0d64f9bf5c7d455793fbf695b2e97623

**Comment by harryguantj** [comment]
https://blog.csdn.net/gyzlhg/article/details/135570244

**Comment by XuandYu000** [comment]
https://www.cnblogs.com/viewoverlooking/p/17963189

**Comment by water2silver** [comment]
https://zhuanlan.zhihu.com/p/677722729

**Comment by BoogalooYue** [comment]
Lesson2 《轻松玩转书生·浦语大模型趣味Demo》
[笔记链接](https://www.yuque.com/g/boogalooyue/knowledge/inql1ryf222dleg0/collaborator/join?token=29BguAW9iPJ0hpnl&source=doc_collaborator#)

**Comment by KyleHuang9** [comment]
https://github.com/KyleHuang9/InternLM-tutorial/blob/main/Lesson2/node.md

**Comment by TonyTnT** [comment]
https://zhuanlan.zhihu.com/p/677757604

**Comment by dddxs** [comment]
https://github.com/dddxs/blog/blob/main/note2%20Demo.md

**Comment by YongXie66** [comment]
作业交晚了，海涵
https://github.com/YongXie66/InternLM_tutorial_copy/blob/main/notes/Lesson2.md

**Comment by BoogalooYue** [comment]
忘记发布了，刚刚重新更新了一下，麻烦了



***@***.***
 
发件人： Mattheliu
发送时间： 2024-01-15 19:06
收件人： InternLM/tutorial
抄送： Yue Shi; Comment
主题： Re: [InternLM/tutorial] 第2节课笔记(21班) (Discussion #222)
demo图片没加上

**Comment by mattheliu** [comment]
https://github.com/mattheliu/gitblog/issues/2
